DQMOffline/Trigger/python/HiggsMonitoring_cff.py

```diff
@@ -1,4 +1,141 @@
+
 import FWCore.ParameterSet.Config as cms
 
+from DQMOffline.Trigger.HiggsMonitoring_cfi import hltHIGmonitoring
+
+##############################DiLepton cross triggers######################################################
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref = hltHIGmonitoring.clone()
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.nmuons = cms.uint32(1)
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.nelectrons = cms.uint32(1)
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.FolderName = cms.string('HLT/Higgs/DiLepton/HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_muref/')
```

Comment:
Hello Mia, 
HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL/mu leg or eleleg makes sense. I named them according to the reference trigger used. For the subdirectory, I followed what other groups did. If you want, I can change to "HLT/DiLepton".